optimum/bettertransformer/models/encoder_models.py

```diff
@@ -17,6 +17,114 @@
 from .base import BetterTransformerBaseLayer
 
 
+class TapasLayerBetterTransformer:
```

Comment:
```suggestion
class TapasLayerBetterTransformer(BetterTransformerBaseLayer):
```

We are calling `super()` in the init so we should inherit from `BetterTransformerBaseLayer`! 👍

Reply:
Thank you for your suggestion! That makes a lot of sense!